webpack.config.js

```diff
@@ -39,36 +27,25 @@ module.exports = function(env, argv) {
                 {
                     test: /\.js$/,
                     include: [
-                        path.resolve(__dirname, "src"),
-                        path.resolve(__dirname, "node_modules/debug")
+                        path.join(__dirname, "src"),
+                        require.resolve("debug")
                     ],
-                    use: {
-                        loader: "babel-loader",
-                        options: {
-                            plugins: ["@babel/plugin-transform-runtime"],
-                            presets: argv.pure ? [] : [
-                                [
-                                    "@babel/preset-env",
-                                    {
-                                        useBuiltIns: "usage",
-                                        modules: "commonjs",
-                                        corejs: {
-                                            version: 3,
-                                            proposals: true
-                                        },
-                                        // debug: true,
-                                        loose: true, // needed for IE 10
-                                    }
-                                ]
-                            ]
-                        }
-                    }
+                    use:  "babel-loader"
```

Comment:
This does not work well for me. It looks like the configuration from `.babelrc.js` is not picked up by `babel-loader` so had to use `use: "babel-loader?configFile=./.babelrc.js"` instead.

Reply:
It definitely works for me. Maybe try using `babel.config.js` instead? Otherwise I'd prefer to see the object syntax, like
```
use: {
    loader: "babel-loader",
    options: {
        configFile: path.join(__dirname, ".babelrc.js")
    }
}
```